Login_SpotifyUI/Login_SpotifyUI/Controller/HomeViewController.swift

```diff
@@ -27,6 +27,9 @@ class HomeViewController: UIViewController {
         
         if let register = self.register {
             homeScrollView.welcomeLabel.text = "환영합니다. \(register.nickName)님"
+            homeScrollView.profileView.playListCountLabel.text = "플레이리스트: \(register.playList?.list.count ?? 0)개"
+            let likeList = register.playList?.list.filter { $0.like == true }
```

Comment:
bool타입의 경우 == true 를 작성할 필요 없습니다.
(false의 경우도 마찬가지 입니다.)
```suggestion
            let likeList = register.playList?.list.filter { $0.like }
```

----

ex1) true인지 판별하고 싶은 경우
```swift
var flag = true

if flag {
   // do something
}
```

ex2) false인지 판별하고 싶은 경우
```swift
var flag = true

if !flag {
   // do something
}
```

Reply:
`register.playList` 가 옵셔널 값이므로 안전하게 언래핑 해주세요~! 
아래 `likeList`도 마찬가지 입니다.